components/automate-ui/src/app/modules/token/token-list/api-token-list.component.scss

```diff
@@ -23,3 +11,14 @@
     }
   }
 }
+
+::ng-deep .mat-select-panel {
+  position: relative;
+  margin-top: 12px;
+  right: 24px;
+  width: 54px;
+}
+
+::ng-deep .mat-select-panel-wrap {
+  margin-left: -75px;
+}
```

Comment:
This was actually how this was set before, but as @msorens mentioned, we can't move this outside the `:host` otherwise we'll 1) possibly unintentionally target `select`s that we don't want to and 2) will end up with the same issue we had before with random ellipsing.

I'm thinking that maybe we move this to a top level so it is only in one single place.

Reply:
@SEAjamieD 
I tried with the different solutions using the `:host` but not working as expected
```
:host {

  .mat-select-panel {
    position: relative;
    margin-top: 12px;
    right: 24px;
    width: 54px;
  }

  .mat-select-panel-wrap {
    margin-left: -75px;
  }
}
```
```
:host .mat-select-panel {
  position: relative;
  margin-top: 12px;
  right: 24px;
  width: 54px;
}

:host .mat-select-panel-wrap {
  margin-left: -75px;
}
```
```
:host(.mat-select-panel) {
  position: relative;
  margin-top: 12px;
  right: 24px;
  width: 54px;
}

:host(.mat-select-panel-wrap) {
  margin-left: -75px;
}
```
its only working with the ng::deep

Reply:
Sorry about the lack of clarity there - It will still need to use `::ng-deep`, however that will need to remain inside `:host` otherwise it will can possibly hit other `.mat-select`s that are not located within this component.

Reply:
@SEAjamieD 
I checked with the HTML part and I found we are using the `penalClass` for this and I test the same thing with the penal class and it's working fine.
we can use 
```
::ng-deep .chef-control-menu {
  position: relative;
  margin-top: 12px;
  right: 24px;
  width: 54px;
}
```
instead of 
```
::ng-deep .mat-select-panel {
  position: relative;
  margin-top: 12px;
  right: 24px;
  width: 54px;
}
```
can we use this?

Reply:
What you're suggesting was causing this tough to reproduce bug here: https://github.com/chef/automate/issues/3983
The update to fix _that_ bug: https://github.com/chef/automate/pull/4178